lib/repository.js

```diff
@@ -653,7 +653,11 @@ Repository.prototype.fetch = function(
         return remote.disconnect();
       });
     })
-    .then(finallyFn, finallyFn);
+    .then(finallyFn)
```

Comment:
Is .finally part of the Node in Promise API? Also, in my case I'd like to catch errors outside of this scope in my own code, would that be affected?